src/ar_tx_blacklist.erl

```diff
@@ -0,0 +1,25 @@
+-module(ar_tx_blacklist).
+-export([load_from_files/1, is_element/2]).
+-include("ar.hrl").
+
+load_from_files(Files) ->
+	sets:from_list(lists:flatten(lists:map(fun load_from_file/1, Files))).
+
+is_element(TX, TXBlacklist) ->
+	sets:is_element(ar_util:encode(TX#tx.id), TXBlacklist).
```

Comment:
Why do we encode the IDs before we put them in the set? I see no reason to do it.

Reply:
We actually simply not decode them before we put them in the set. Having them encoded in the file is convenient cause people can make sense of them. What do you think?

Reply:
We could decode them before putting in the set to make it smaller though..

Reply:
✅